package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rdvue",
-  "version": "0.5.5",
+  "version": "0.5.6",
```

Comment:
Prob best to not bump the version until you're ready to deploy otherwise we'll be at 10K soon, lol.